js/src/redux/providers/status.js

```diff
@@ -48,6 +48,15 @@ export default class Status {
         }
 
         this._store.dispatch(statusBlockNumber(blockNumber));
+
+        this._api.eth
+          .getBlockByNumber(blockNumber)
+          .then((block) => {
+            this._store.dispatch(statusCollection({ gasLimit: block.gasLimit }));
```

Comment:
Should we store any other informations back from this request ? I know we are fetching timestamps for transactions from the blockNumber, we might want to store it.. Or not... It should take only 50kB of memory for one day of data, might be worth it